### PR TITLE
Fix broken link to Windows 10 SDK

### LINKED ISSUE #10099: Windows SDK 10 link is broken in NuGet.Client Contribution guide
## Details about Problem

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe): -

NuGet version (x.x.x.xxx): -

dotnet.exe --version (if appropriate): -

VS version (if appropriate): -

OS version (i.e. win10 v1607 (14393.321)): -

Worked before? If so, with which NuGet version: -

## Detailed repro steps so we can see the same problem

1. Click link
2. Link doesn't go to Windows 10 SDK download page

## Other suggested things

I submitted a PR to fix here https://github.com/NuGet/NuGet.Client/pull/3697

### Verbose Logs

Please include verbose logs (NuGet.exe <COMMAND> -verbosity detailed | dotnet.exe <COMMAND> --verbosity diag | etc...) -

### Sample Project

Very helpful if you can zip a project and paste into this issue! -

### PR DESCRIPTION
**Pull request by bjorkstromm**
## Bug

Fixes: https://github.com/NuGet/Home/issues/10099
Regression: No  
* Last working version:  -
* How are we preventing it in future:  -

## Fix

Details: This fixes a broken link in contributing documentation.

## Testing/Validation

Tests Added: No  
Reason for not adding tests:  Only documentation
Validation:  -

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
   - .NET desktop development
   - Visual Studio extension development.
   - Desktop development with C++
-- [Windows 10 SDK](https://dev.windows.com/en-US/downloads/windows-10-sdk)
+- [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk/)
```
Comment (nkolev92) [inline CONTRIBUTING.md]: I'm not if this is still needed for building. I think it's needed if you want to run all *E2E* tests though. 

I created a follow up. https://github.com/NuGet/Home/issues/10096

**Review by nkolev92 — APPROVED** [review]
Thanks @mholo65

**Comment by nkolev92** [comment]
ah, I forgot, can you please create a tracking issue for this @mholo65 

We want to have every commit correlated to an issue.

**Comment by bjorkstromm** [comment]
@nkolev92 thanks! Submitted issue and updated description.

File: CONTRIBUTING.md (hunk 1 above)
Comment (campersau) [inline CONTRIBUTING.md]: Would it be better to not link to the language specific version?
```suggestion
- [Windows 10 SDK](https://developer.microsoft.com/windows/downloads/windows-10-sdk/)
```

Comment (zivkan) [reply CONTRIBUTING.md]: Although this is not part of Microsoft's doc's site, this is the recommendation for docs: https://docs.microsoft.com/en-us/contribute/how-to-write-links#links-to-pages-that-are-not-part-of-the-technical-documentation-set

@mholo65 can you please commit this suggestion.

**Review by zivkan — APPROVED** [review]

**Comment by zivkan** [comment]
@mholo65 This branch also needs to be rebased onto the latest dev.

**Comment by bjorkstromm** [comment]
@zivkan rebased.

**Comment by zivkan** [comment]
This must have been the unluckiest PR we ever had. I swear I had to run about 10-15 builds before there was one without flaky failures. Finally a green build 🥳 .  @mholo65 thanks for your contribution.

**Pull request merged**